packages/react-atlas-core/src/avatar/Avatar.js

```diff
@@ -80,7 +115,7 @@ Avatar.styleguide = {
     {/* icon beats title */}
     <Avatar title="Nathan" icon={<i className="fa fa-github"></i>} />
     {/* image beats icon */}
-    <Avatar 
+    <Avatar
       icon={<i className="fa fa-github"></i>}
       image="cat.jpg"
     />
```

Comment:
Looks like this commit also include the Avatar changes? These should be different branches and not conflict with each other - hopefully.